bin/tronfig

```diff
@@ -14,32 +16,24 @@ from tron.config import schema, manager, ConfigError
 log = logging.getLogger('tronfig')
 
 
-def set_options_from_args(options, args):
-    if args and args[-1] == '-':
-        options.from_stdin = True
-        args = args[:-1]
-    else:
-        options.from_stdin = False
-
-    options.config_name = args[0] if args else schema.MASTER_NAMESPACE
-
-
 def parse_options():
-    usage  = "usage: %prog [<name>] [options] [-]"
+    usage  = "usage: %prog [options] [<name>] [-|path]"
     parser = cmd_utils.build_option_parser(usage)
 
     parser.add_option("-p", "--print", action="store_true", dest="print_config",
                       help="Print config to stdout, rather than uploading")
-    parser.add_option("-n", "--no-header", action="store_true", default=False,
-                      help="Print named config without a header")
     parser.add_option("-C", "--check", action="store_true", dest="check",
-                      help="Only check configuration, don't apply")
+                      help="Upload and check configuration, don't apply")
```

Comment:
I think it would be helpful to say when you might want to upload, vs not.
Looks like uploading is good for checking one namespace against the current trond configuration, whereas -v just checks that namespace fragment?